server/src/internalClusterTest/java/org/opensearch/gateway/remote/RemoteClusterStateCleanupManagerIT.java

```diff
@@ -108,6 +129,23 @@ public void testRemoteCleanupDeleteStale() throws Exception {
             .add("cluster-state")
             .add(getClusterState().metadata().clusterUUID());
         BlobPath manifestContainerPath = baseMetadataPath.add("manifest");
+
+        BlobStoreRepository routingTableRepository = (BlobStoreRepository) repositoriesService.repository(REPOSITORY_3_NAME);
+        List<IndexRoutingTable> indexRoutingTables = new ArrayList<>(getClusterState().routingTable().indicesRouting().values());
+        BlobPath indexRoutingBasePath = baseMetadataPath.add(INDEX_ROUTING_PATH_TOKEN);
+        RemoteStoreEnums.PathHashAlgorithm pathHashAlgo = RemoteStoreEnums.PathHashAlgorithm.FNV_1A_BASE64;
+        BlobPath indexRoutingPath = pathType.path(
+            RemoteStorePathStrategy.PathInput.builder()
+                .basePath(indexRoutingBasePath)
+                .indexUUID(indexRoutingTables.get(0).getIndex().getUUID())
+                .build(),
+            pathHashAlgo
+        );
+        assertBusy(() -> {
+            // There would be >=3 files as shards will transition from INIT -> UNASSIGNED -> STARTED state
```

Comment:
nit: order is wrong in comment. correct order for starting shards is UNASSIGNED -> INIT  -> STARTED